### ISSUE TITLE
mce_loop_s passes chunk_ref as a scalar rather than as a reference if there is only one element in the sequence

### ISSUE THREAD

**Issue by djerius** [marioroy/mce-perl#10]
The following code:

```
use MCE::Loop;
use Data::Dumper;

mce_loop_s {
    my ($mce, $chunk_ref, $chunk_id) = @_;
    print Dumper { chunk_ref => $chunk_ref, id => $chunk_id };
} 1,1;
```

Produces
```
$VAR1 = {
          'chunk_ref' => 1,
          'id' => 1
        };
```
I expected it to produce something like this:

```
$VAR1 = {
          'chunk_ref' => [ 1 ],
          'id' => 1
        };
```

Thanks!

**Comment by marioroy** [comment]
Thank you for the bug report. It's a bug, because MCE models never compute below 2 for the chunk_size value on chunk_size => 'auto'. It should of passed an array ref even though the sequence data contains 1 element.

```
print MCE->chunk_size(), "\n"
```

I will make a fix and report back.

**Comment by marioroy** [comment]
Hi @djerius. I released [MCE 1.833](https://metacpan.org/release/MARIOROY/MCE-1.833) containing the fix.

Thanks again.

**Comment by djerius** [comment]
Thanks!
